src/app/withdraw/components/withdraw-amount.tsx

```diff
@@ -5,10 +5,12 @@ import { Schema } from "yup";
 export const WithdrawFlowAmount = ({
   validationSchema,
   handleSubmit,
+  maxFee,
   disabled,
 }: {
   validationSchema: Schema;
   handleSubmit: (value: any) => void;
+  maxFee: number;
```

Comment:
`maxFee` can be undefined (we lie to the compiler with `maxFee!`, but we set `disabled` to `maxFee === undefined` so it's a possibility) and in that case we would get a `NaN` in the fee computation as it's not guarded by `disabled`. Am I missing something?